spine-c/src/spine/MeshAttachment.c

```diff
@@ -76,23 +75,69 @@ void spMeshAttachment_updateUVs (spMeshAttachment* self) {
 
 void spMeshAttachment_computeWorldVertices (spMeshAttachment* self, spSlot* slot, float* worldVertices) {
```

Comment:
spMeshAttachment should use spVertexAttachment `computeWorldVertices`. spine-libgdx doesn't only because there meshes store their verts in a libgdx specific data structure to avoid a data copy. For generic runtimes this isn't possible.